docs/content/guides/upgrade-and-migration/release-notes.md

```diff
@@ -25,7 +25,7 @@ See the full history of changes made to Handsontable in each major, minor, and p
 
 Released on 22nd of September, 2022 - [view docs](docs/12.1/)
 
-### Fixed
+#### Fixed
```

Comment:
The header type is already changed on the release branch https://github.com/handsontable/handsontable/blame/release/12.2.0-docs/docs/content/guides/upgrade-and-migration/release-notes.md#L52. I'd consider to undo the change to avoid conflicts 🤔 WDYT?

Reply:
As @krzysztofspilka's unavailable, I'll make this change 👍.